stor/tests/test_integration_dx.py

```diff
@@ -0,0 +1,320 @@
+import gzip
+import os
+import pytest
+import time
+import unittest
+from unittest import skipIf
+
+import dxpy
+import mock
+import six
+
+from stor import exceptions
+from stor import NamedTemporaryDirectory
+from stor import Path
+from stor.dx import DXPath
+from stor.test import DXTestCase
+from stor.tests.shared import assert_same_data
+from stor.tests.test_integration import BaseIntegrationTest
+import stor
+import stor.tests.test_integration as test_integration
+
+
+class DXIntegrationTest(BaseIntegrationTest.BaseTestCases, DXTestCase):
+    def setUp(self):
+        self.vcr_enabled = False  # don't want vcr playback for integration tests
+        super(DXIntegrationTest, self).setUp()
+        if not os.environ.get('DX_AUTH_TOKEN'):
+            raise unittest.SkipTest(
+                'DX_AUTH_TOKEN env var not set. Skipping integration test')
+
+        self.setup_temporary_project()
+        self.project = DXPath('dx://' + self.project + ':')
+        self.test_dir = self.project / 'test'
+
+    def tearDown(self):
+        super(DXIntegrationTest, self).tearDown()
+
+    def test_login_auth(self):
```

Comment:
what is this function testing? I'm very unclear

Reply:
This function is mocking out the internal `SECURITY_CONTEXT` token used by dxpy, which is actually set in `dxpy.AUTH_HELPER` during login. This function is then checking if mocking that out raises an error for dxpy calls, while keeping it intact allows dxpy operations.

Reply:
needs a different name then - I guess `test_mocking_login_auth_works` (??)

Reply:
FIxed.